orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java

```diff
@@ -104,9 +109,9 @@ public InstanceStatusResponse getInstance(@PathParam("instanceId") String instan
     }
 
     @GET
-    @Path("/{instanceId}/serviceStatus")
+    @Path("/{instanceId}/serviceStatusInfo")
```

Comment:
Is it safe to remove the old path? Will nobody use the old path during an upgrade?

Reply:
There are no references to "/serviceStatus" elsewhere in the code.

This is a fairly recent endpoint that only a few people know of and is intended for debugging.

Reply:
👍